samples/ingestion/ingestion-client/StartTranscriptionByServiceBus/Program.cs

```diff
@@ -18,8 +17,7 @@ public static class Program
         /// Main entry point of the function app.
         /// </summary>
         /// <param name="args"></param>
-        /// <returns>A <see cref="Task"/> representing the result of the asynchronous operation.</returns>
-        public static async Task Main(string[] args)
+        public static void Main(string[] args)
```

Comment:
Any reason why we are not going async now?

Reply:
no - I was just following the sample apps while trying to figure out why it does not work and moved it to sync while doing so.

I don't think that it's necessary, but kept it in my PR anyway..